descriptors/ModuleDescriptor-template.json

```diff
@@ -475,6 +475,26 @@
       } ]
     }
   ],
+  "requires": [
+    {
+      "id": "configuration",
+      "version": "2.0"
+    },
+    {
+      "id": "users",
+      "version": "16.0"
+    },
+    {
+      "id": "email",
+      "version": "1.0"
+    }
+  ],
+  "optional": [
+    {
+      "id": "orders",
```

Comment:
added as optional, because mod-orders already declared dependency to mod-finance-storage and so to avoid circular dependency
https://github.com/folio-org/mod-orders/blob/master/descriptors/ModuleDescriptor-template.json#L1298
https://github.com/folio-org/mod-orders/blob/master/descriptors/ModuleDescriptor-template.json#L1334